src/engine/renderer/tr_local.h

```diff
@@ -200,6 +215,23 @@ static inline void halfToFloat( const f16vec4_t in, vec4_t out )
 // max. 16 dynamic lights per plane
 #define LIGHT_PLANES ( MAX_REF_LIGHTS / 16 )
 
+struct glVertexShim_t
+{
+	GLint floatFormat;
+};
+
+extern glVertexShim_t GL_vertexShim;
+
+static inline void glVertexSetHalfFloat()
+{
+	GL_vertexShim.floatFormat = GL_HALF_FLOAT;
+}
+
+static inline void glVertexSetFloat()
+{
+	GL_vertexShim.floatFormat = GL_FLOAT;
```

Comment:
These are GLenum, which is an unsigned int, as opposed to GLint, which is just int. I guess it doesn't matter here, since all values happen to be positive, but better to use the correct type.

Reply:
Right.